### DISCUSSION TITLE
Achievements

### DISCUSSION THREAD

**Discussion by ababyprogrammer** [community/community#113950] [Discussions]
### Select Topic Area

General

### Hello to all

I love _Achievements_ in my profile.

Can any body tell me how to give more of them? Like this user @dmalan.

Thank you.

[A baby programmer](https://github.com/ababyprogrammer)

**Comment by ArifHussain39** [comment]
Here is youtube video for github badges.
https://www.youtube.com/watch?v=oJWFfp3sGYA

**Comment by devashish2024** [comment]
I'd recommend you checkout Thinkright20's Profile-Badges repo, it has a lot information and simplified. Check out [here](https://github.com/Thinkright20/Profile-Badges)

Consider marking as answer if this solves your question.

**Comment by queenofcorgis** [comment]
Hi @ababyprogrammer . 

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).
